index.html

```diff
@@ -22,21 +22,120 @@ <h1>Lucky Duck Restaurang</h1>
   <div class="allergens-icon"></div>
 
  </div><!--End of Navbar-->
- <!------------------------------------------------------------------>
+  <!------------------------------------------------------------------>
+  
  <div class="menu-component">
-  <div class="meal-1">
-
+  <!--add more meal from here, just copy this div -->
+  <!------------------meal1----------------->
+  <div class="meal">
+   <div class="image-meal">
+    <img src="./img/FLÄSKFILÉPLANKA.jpg" alt="FLÄSKFILÉPLANKA">
+   </div>
+   <div class="hover-text">
```

Comment:
This menu is supposed to be viewed on an iPad. Requiring them to click the item to show it is not a good user experience. 

Also, the items are too big to be shown on an iPad. It ends up getting clipped 
![image](https://user-images.githubusercontent.com/36706634/195836075-d49446cd-fff6-4784-a1bc-fe84e99762b4.png)

Reply:
I do also think we decided against using images, since it doesn't have a consistent brand across the different images.